mlflow/genai/scorers/base.py

```diff
@@ -532,6 +533,18 @@ def custom_length_check(outputs) -> bool:
             if new_scorer._cached_dump is not None:
                 new_scorer._cached_dump["name"] = name
 
+        # Check if zerobus is enabled and require SQL warehouse ID
+        if self._is_zerobus_enabled(experiment_id):
+            if MLFLOW_TRACING_SQL_WAREHOUSE_ID.get() is None:
+                raise MlflowException.invalid_parameter_value(
+                    "MLFLOW_TRACING_SQL_WAREHOUSE_ID environment variable must be set when "
+                    "registering a scorer for an experiment with zerobus enabled "
+                    "(mlflow.experiment.databricksTraceStorageTable tag present)."
```

Comment:
```suggestion
                    "MLFLOW_TRACING_SQL_WAREHOUSE_ID environment variable must be set when "
                    "registering a scorer for an experiment linked to a Unity Catalog trace table.
```

Let's not surface internal details in customer-facing message.